node_modules/oae-admin/importusers/js/importusers.js

```diff
@@ -166,9 +166,8 @@ define(['jquery', 'oae.core', 'jquery.fileupload', 'jquery.iframe-transport'], f
                     'backdrop': 'static'
                 });
 
-                // Hide the spinner icon using jQuery
-                // @see https://github.com/FortAwesome/Font-Awesome/issues/729
```

Comment:
But I do notice that the reference wasn't removed in some other files. I'll remove it from them.